app/main.py

```diff
@@ -1 +1,6 @@
-# write your code here
+def copy_file(command: str) -> None:
+    file_r = command.split(" ")[1]
+    file_w = command.split(" ")[2]
```

Comment:
Don't call split twice. You can create separate variable for storing result of `split ` and use if where you need it. Also, `" "` is the default value for split
```suggestion
    file_r = command.split()[1]
    file_w = command.split()[2]
```